openapi/spec.json

```diff
@@ -1425,7 +1426,8 @@
           }
         },
         "required": [
-          "project_id"
+          "project_id",
+          "name"
```

Comment:
This seems wrong. `name` should be optional for a new experiment.

Reply:
Done!